Tests/CryptoTests/Authenticated Encryption/ChaChaPoly-Runner.swift

```diff
@@ -66,7 +66,7 @@ class ChaChaPolyTests: XCTestCase {
         let ciphertext = try! ChaChaPoly.seal(plaintext, using: key, nonce: nonce, authenticating: Data())
```

Comment:
I think we should change the signature if this function to be throwing, and remove the `!` from `try!`. As seen in many of the `AESGCMTests` tests.